src/locale/locales/es/messages.po

```diff
@@ -407,7 +407,7 @@ msgstr "Has dejado de seguir a esta cuenta"
 
 #: src/view/com/profile/ProfileMenu.tsx:98
 msgid "Account unmuted"
-msgstr "Cuenta demuteada"
+msgstr "Cuenta con sonido reactivado"
```

Comment:
I kinda vote for...
`Account muted` -> `Cuenta silenciada`
`Account unmuted` -> `Cuenta no silenciada`

Feels more consitent than `sonido reactivado` (sound reactivated doesn't seem fitting, as there's no sound involved here)

Reply:
Fixed as voted